interactions/models/discord/auto_mod.py

```diff
@@ -97,16 +105,22 @@ def _keyword_converter(filter: str | list[str]) -> list[str]:
 class KeywordTrigger(BaseTrigger):
     """A trigger that checks if content contains words from a user defined list of keywords"""
 
-    type: AutoModTriggerType = attrs.field(
-        default=AutoModTriggerType.KEYWORD,
-        converter=AutoModTriggerType,
```

Comment:
Why are you removing the type declarations from all the triggers?

Reply:
It's already defined in the BaseTrigger class. The only thing that you lose is the default value, but since type should always be passed I don't see how that's an issue. Plus, some Triggers and Actions actually did not have it re-declared.

Reply:
It's intentional. Don't remove them

Reply:
> It's intentional. Don't remove them

Sure I can add them back, but then shouldn't all classes have it? Why, for example, `MentionSpamTrigger` didn't have one?

Reply:
In case you haven't noticed the automod implementation is second rate. 

As for why type is explicitly defined here, it was at request

Either way, please don't remove them - feel free to draft a complete rewrite ahead for V6, but not in this PR